kapitan/secrets.py

```diff
@@ -433,6 +432,24 @@ def rsa_private_key(key_size=''):
     ), "UTF-8")
 
 
+def rsa_public_key(secrets_path, private_key_file):
+    """derives an RSA public key from private_key_file"""
+    token = "gpg:%s" % private_key_file
```

Comment:
Nit: We could use the new string interpolation format `"gpg:{}".format(private_key_file)`